adam-cli/src/main/scala/org/bdgenomics/adam/cli/Transform.scala

```diff
@@ -84,6 +84,8 @@ class TransformArgs extends Args4jBase with ADAMSaveAnyArgs with ParquetArgs {
   var forceLoadParquet: Boolean = false
   @Args4jOption(required = false, name = "-single", usage = "Saves OUTPUT as single file")
   var asSingleFile: Boolean = false
+  @Args4jOption(required = false, name = "-filename_record_group", usage = "Set converted FASTQs' record-group names to this value; if empty-string is passed, use the basename of the input file, minus the extension.")
```

Comment:
I feel like this would be a bit cleaner as two switches:

* One that is "set record group ID from filename"
* One that is "set record group ID to passed in string"

They would be mutually exclusive. A "set sample ID" flag would be useful as well, IMO. I defer to you as to whether you'd like to implement that or not.